src/catalogue/category/catalogueCardView.component.tsx

```diff
@@ -51,25 +51,19 @@ function CatalogueCardView(props: CatalogueCardViewProps) {
     endIndex
   );
 
+  const cardViewHeight = getPageHeightCalc('100px');
```

Comment:
I am not sure this works correctly as if you decrease the window width the whole bottom section moves to the right:
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/32da0d2e-7b66-46b6-bd21-a526a2d9aabf)
setting this to 0 made it work, although it is not inline
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/818fe6d5-2609-4c58-8e6d-197744fdbd32)

Reply:
On both scigateway and just localhost300 it works fine no matter how small I make the screen. How exactly did you get it to be to the right?

Reply:
Just by decreasing the width of the screen. Perhaps the catalogue categories need to be filled with enough to create a scroll bar?
![rowTotalBug](https://github.com/ral-facilities/inventory-management-system/assets/90245114/4a1e88d0-ef75-43b8-8ccc-ef3c36892af7)